### PR TITLE
core-depends: add podman and recommended packages

### PR DESCRIPTION
**Pull request by ramcq**
Add podman to the OS image, including slirp4netns and uidmap which are required
for using podman as an unprivileged user.

https://phabricator.endlessm.com/T25542

### CODE REVIEW

File: eos-core-depends [hunk 1]
```diff
@@ -172,6 +172,7 @@ openprinting-ppds
 openssh-server
 # For flatpak-builder
 patch
+podman
```
Comment (wjt) [inline eos-core-depends]: Won't you also want to include `podman-docker` once that's in the distro?

Comment (wjt) [reply eos-core-depends]: Resolution: no, because without `docker-compose` support, having `/usr/bin/docker` provides false hope.

**Comment by wjt** [comment]
How about fuse-overlayfs?

**Comment by ramcq** [comment]
> How about fuse-overlayfs?

Voila. Ready to roll.

**Comment by wjt** [comment]
It'd be nice if this package had enough of a build system that it could check if all the packages are actually present in the distro, eh?

File: eos-core-depends [hunk 2]
```diff
@@ -101,6 +101,7 @@ fonts-vlgothic
 fonts-wqy-zenhei
 foomatic-db-compressed-ppds
 fprintd
+fuse-overlayfs
```
Comment (wjt) [inline eos-core-depends]: I think you want `# For rootless podman` here?

Comment (ramcq) [reply eos-core-depends]: Done!

**Pull request merged**